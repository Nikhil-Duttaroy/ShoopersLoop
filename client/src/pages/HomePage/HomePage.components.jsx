import React from "react";
// import './HomePage.styles.scss';
import Directory from './../../components/Directory/Directory.components';
import {HomePageContainer} from './HomePage.styles';
import AliceCarousel from "react-alice-carousel";
import "react-alice-carousel/lib/alice-carousel.css";
import Footer from './../../components/Footer/Footer.component';

const handleDragStart = (e) => e.preventDefault();

const items = [
  <img
    src='https://i.ibb.co/wNWVMCJ/wardrobe-1.jpg'
    alt="1"
    onDragStart={handleDragStart}
    style={{ width: "100%", height: "70vh", objectFit: "cover" }}
  />,
  <img
    alt='2'
    src='https://i.ibb.co/C9sw3F8/overview.jpg'
    onDragStart={handleDragStart}
    style={{ width: "100%", height: "70vh", objectFit: "cover" }}
  />,
];

  const HomePage =() =>{
  return (
    <HomePageContainer>
      <AliceCarousel
        mouseTracking
        items={items}
        autoPlay='true'
        animationDuration='5000'
        disableDotsControls='true'
        infinite='true'
      />
      <Directory  />
      {/* <Footer/> */}
    </HomePageContainer>
  );
  }

export default HomePage