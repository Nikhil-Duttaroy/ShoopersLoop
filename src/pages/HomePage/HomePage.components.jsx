import React from "react";
import './HomePage.styles.scss';
import Directory from './../../components/Directory/Directory.components';

  const HomePage =() =>(
    <div className='homepage'>
      <Directory/>
    </div>
  )

export default HomePage