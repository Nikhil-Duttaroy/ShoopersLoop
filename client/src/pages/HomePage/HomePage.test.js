import React from "react";
import { shallow } from "enzyme";
import HomePage from './HomePage.components';

it("should render Homepage component", () => {
  expect(shallow(<HomePage />)).toMatchSnapshot();
});
