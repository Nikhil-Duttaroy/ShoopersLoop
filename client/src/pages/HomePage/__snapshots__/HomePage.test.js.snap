// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Homepage component 1`] = `
<Fragment>
  <Helmet
    defer={true}
    encodeSpecialCharacters={true}
    prioritizeSeoTags={false}
  >
    <title>
      ShoopersLoop : Voice Controlled Ecommerce App
    </title>
    <meta
      content="ShoopersLoop : Voice Controlled Ecommerce App , Homepage for shoopersloop"
      name="description"
    />
  </Helmet>
  <styled.div>
    <t
      activeIndex={0}
      animationDuration="5000"
      animationEasingFunction="ease"
      animationType="slide"
      autoHeight={false}
      autoPlay="true"
      autoPlayControls={false}
      autoPlayDirection="ltr"
      autoPlayInterval={400}
      autoPlayStrategy="default"
      autoWidth={false}
      controlsStrategy="default"
      disableButtonsControls={false}
      disableDotsControls="true"
      disableSlideInfo={true}
      infinite="true"
      innerWidth={0}
      items={
        Array [
          <img
            alt="1"
            onDragStart={[Function]}
            src="https://i.ibb.co/wNWVMCJ/wardrobe-1.jpg"
            style={
              Object {
                "height": "70vh",
                "objectFit": "cover",
                "width": "100%",
              }
            }
          />,
          <img
            alt="2"
            onDragStart={[Function]}
            src="https://i.ibb.co/C9sw3F8/overview.jpg"
            style={
              Object {
                "height": "70vh",
                "objectFit": "cover",
                "width": "100%",
              }
            }
          />,
        ]
      }
      keyboardNavigation={false}
      mouseTracking={true}
      name=""
      onInitialized={[Function]}
      onResized={[Function]}
      onSlideChange={[Function]}
      onSlideChanged={[Function]}
      paddingLeft={0}
      paddingRight={0}
      ssrSilentMode={true}
      swipeDelta={20}
      swipeExtraPadding={200}
      touchMoveDefaultEvents={true}
      touchTracking={true}
    />
    <Connect(Directory) />
  </styled.div>
</Fragment>
`;
